RunLog/RunLog/Sources/Presentation/Log/View/LogViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  LogViewController.swift
+//  RunLog
+//
+//  Created by 신승재 on 3/14/25.
+//
+
+import UIKit
+import Combine
+import SnapKit
+import Then
+
+final class LogViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let logView = LogView()
+    private let logViewModel = LogViewModel()
+    
+    private var dataViewControllers: [UIViewController] = []
+    private var currentPage: Int = 0
+    private var cancellables = Set<AnyCancellable>()
+
+    // MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupNavigationBar()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: animated)
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+
+    // MARK: - Setup UI
+    private func setupUI() {
+        dataViewControllers.append(
+            CalendarViewController(viewModel: logViewModel)
+        )
+        
+        dataViewControllers.append(
+            TimelineViewController(viewModel: logViewModel)
+        )
+        
+        view.addSubview(logView)
+        logView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+        
+        logView.segmentedControl.addTarget(
+            self,
+            action: #selector(changeValue(control:)),
+            for: .valueChanged
+        )
+        
+        logView.pageViewController.setViewControllers(
+            [dataViewControllers[0]],
+            direction: .forward,
+            animated: true
+        )
+        
+        logView.pageViewController.delegate = self
+        logView.pageViewController.dataSource = self
+    }
+
+    // MARK: - Setup Navigation Bar
+    private func setupNavigationBar() {
+        title = "LOGO"
```

Comment:
```suggestion
        navigationItem.title = "LOGO"
```
self.title의 이름을 바꾸면 탭바 아이템에서 이름이 덮어쓰여져 버립니다!

Reply:
Logo가 들어갈 예정이라서 고냥 지워놓도록 하겠습니다 !